3.10/deployment-cluster-manual-start.md

```diff
@@ -181,7 +187,10 @@ arangod --server.endpoint tcp://0.0.0.0:8531 \
   --agency.activate true \
   --agency.size 3 \
   --agency.supervision true \
-  --database.directory agent 
+  --agency.endpoint tcp://192.168.1.1:8531 \
+  --agency.endpoint tcp://192.168.1.2:8531 \
+  --agency.endpoint tcp://192.168.1.3:8531 \
```

Comment:
Strictly speaking, this is not necessary, AFAIK. The first agent can be started without knowing others.

Reply:
K I'll update that.